src/components/Stake/Stake.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Header } from '@aragon/ui'
+import LayoutBox from '../Layout/LayoutBox'
+import LayoutColumns from '../Layout/LayoutColumns'
+import LayoutGutter from '../Layout/LayoutGutter'
+import LayoutLimiter from '../Layout/LayoutLimiter'
+import SideBar from './SideBar'
+
+const Stake = React.memo(function Stake() {
+  return (
+    <LayoutGutter
+      css={`
+        position: relative;
+        z-index: 2;
+      `}
+    >
+      <LayoutLimiter>
+        <Header primary="Stake management" />
+        <LayoutColumns
+          primary={<LayoutBox primary />}
+          secondary={<SideBar />}
+        />
```

Comment:
I've added an issue for an improvement to allow us to invert the columns [here](https://github.com/aragon/network-dashboard/issues/90) feel free to add that improvement separately if you like!

Reply:
yes! I was planning to in the next stake management PR :)